bolt/test/X86/Inputs/dwarf5-subprogram-multiple-ranges-other.s

```diff
@@ -0,0 +1,385 @@
+# clang++ -fbasic-block-sections=all -ffunction-sections -g2 -gdwarf-5
+# int doStuffOther(int val) {
+#   if (val)
+#     ++val;
+#   return val;
+# }
+#
+# int mainOther(int argc, const char** argv) {
```

Comment:
Are you sure your cpp actually contained mainOther?

Reply:
Regenerated .s to make sure it's the correct ASM.